### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by Julow**
also the build was broken

### CODE REVIEW

File: dune-project [hunk 1]
```diff
@@ -77,4 +77,11 @@
    (>= 3.11.0))
   (js_of_ocaml-toplevel
    (>= 3.11.0))
-  (alcotest :with-test)))
+  (alcotest :with-test)
+  ; hilite
+  textmate-language
+  ; tools/ood-gen
```
Comment (tmattio) [inline dune-project:83]: These are listed in the Makefile in the `DEV_DEPS` variable. The reason for that is: they are not dependencies of the project, they are only used to re-generate the data modules. So typically, we don't want them to be installed in Docker.

Comment (Julow) [reply dune-project:83]: Perhaps these dependencies can be marked `{dev}` then.

Comment (Julow) [reply dune-project:83]: I think `with-test` makes more sense, I've updated the PR.

Comment (tmattio) [reply dune-project:83]: Cool, happy to have them with the `with-test` flag 🙂 
Can you also add the complete list of dependencies:
```
ppx_deriving_yaml
ezjsonm
ptime
fmt
fpath
piaf
lambdasoup
cmdliner
crunch
xmlm
uri
```

Comment (Julow) [reply dune-project:83]: Updated!

**Comment by tmattio** [comment]
Thanks! 🙂

**Pull request merged**